### PR TITLE
pagination added for v2/bids

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**

### CODE REVIEW

**Comment by mjoyce91** [comment]
Getting empty response

Screen Shot 2022-03-18 at 11 25 57 AM

**Comment by elizabeth-jimenez** [comment]
> Getting empty response
> 
> Screen Shot 2022-03-18 at 11 25 57 AM

Yea, cause im requiring a perdet - like how v1/bids used to - it was a discussion/tbd, will check back in with Scott

File: src/services/common.js [hunk 1]
```diff
@@ -419,6 +419,7 @@ const bidNameMapping = (val, toWS=false) => {
     posnumtext: 'position',
     posorgshortdesc: 'short_desc',
     postitledesc: 'pos_title_desc',
+    ubwperdetseqnum: 'perdet_seq_num',
```
Comment (mjoyce91) [inline src/services/common.js:422]: Is this field supposed to be coming through in the response? Not seeing it

Screen Shot 2022-03-21 at 4 33 07 PM

Comment (mjoyce91) [reply src/services/common.js:422]: Should clarify - it doesn't come through in DEV1 either, was just curious that all the other fields match up except this one. Is it just used to query the data in the first place?

Comment (elizabeth-jimenez) [reply src/services/common.js:422]: Yea, exactly!

Comment (elizabeth-jimenez) [reply src/services/common.js:422]: so `perdet_seq_num` is the TM name, would have to filter by WS version `ubwperdetseqnum`

Comment (elizabeth-jimenez) [reply src/services/common.js:422]: and for context, i did start to map all of the possible data from WS for all EPs, but i gave up after 1 EP 😂  - now i'll just add what is needed, as we need it

**Review by mjoyce91 — APPROVED** [review]

**Pull request merged**